rq/worker.py

```diff
@@ -591,6 +591,10 @@ def dequeue_job_and_maintain_ttl(self, timeout):
                 break
             except DequeueTimeout:
                 pass
+            except redis.exceptions.ConnectionError as conn_err:
+                self.log.error('Could not connect to Redis instance: %s '
+                               'Retrying...', conn_err)
+                time.sleep(1)
```

Comment:
maybe would be better to use an exponential backoff approach here instead of trying each 1 second?

Reply:
Ok, added